tests/decorators/tests.py

```diff
@@ -291,6 +313,44 @@ def method(self):
 
         self.assertTrue(Test().method())
 
+    def test_iterable_of_decorators(self):
+        """
+        @method_decorator can accept an iterable of decorators.
+        """
+        def add_question_mark(func):
+            def _wrapper(*args, **kwargs):
+                return func(*args, **kwargs) + "?"
+            return _wrapper
+
+        def add_exclamation_mark(func):
+            def _wrapper(*args, **kwargs):
+                return func(*args, **kwargs) + "!"
+            return _wrapper
+
+        # the order should be coherent with the
+        # classical criterion of decorators application
+        # >>> @add_exclamation_mark
+        # >>> @add_question_mark
+        # >>> def func():
+        # ...    return ""
+        decorators = (
```

Comment:
Since decorator ordering is important, I'd add a test for `set` (it probably should raise a TypeError for this case).